sources/id/indowebnovel.py

```diff
@@ -46,12 +46,12 @@ def read_novel_info(self):
                     "id": chap_id,
                     "volume": vol_id,
                     "url": self.absolute_url(a["href"]),
-                    "title": a.text.strip() or ("Chapter %d" % chap_id),
+                    "title": a.get_text(" - ").strip() or ("Chapter %d" % chap_id),
```

Comment:
This can be used: `a.get_text(" - ", strip=True)`

Reply:
Please send pull request to `dev` branch

Reply:
I will create a new PR, because I previously took it from the main branch.

Reply:
I created a new PR here
https://github.com/dipu-bd/lightnovel-crawler/pull/1949/files

Reply:
thanks